lib/bn.js

```diff
@@ -459,6 +460,11 @@ BN.prototype.toJSON = function toJSON() {
   return this.toString(16);
 };
 
+BN.prototype.toBuffer = function toBuffer(endian, length) {
```

Comment:
Since `semistandard` is used now, the above will fail with: `bn.js:510:19: A constructor name should not start with a lowercase letter.` The parameter must be named `Cons` to make the `new` keyword happy.